package/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the packages/archlinux/PKGBUILD file by mosra <mosra@centrum.cz> and the MSYS2/MinGW PKGBUILD templates
+
+_realname=corrade
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=dev
+pkgrel=1
+pkgdesc="C++11/C++14 multiplatform utility library"
+arch=('any')
+url="https://magnum.graphics/corrade/"
+license=('MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" "${MINGW_PACKAGE_PREFIX}-ninja")
+options=(!strip)
+provides=('corrade-git')
+
+_rootdir=$startdir/../../
```

Comment:
Can you put both files into `package/msys/` instead of directly in `package/`? Looks like this `_rootdir` variable is even expecting that :)